var/spack/repos/builtin/packages/py-reportlab/package.py

```diff
@@ -19,3 +19,8 @@ class PyReportlab(PythonPackage):
     # - easy_install, provided by py-setuptools
     # - pip, provided by py-pip
     extends('python', ignore=r'bin/.*')
+
+    depends_on('py-setuptools@2.2:', type='build')
+    depends_on('py-pip@1.4.1:', type='build')
+    depends_on('py-pillow@2.4.0:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('pil@2.4.0:', type=('build', 'run'))
```
We have a virtual dep for this

Reply:
I was not aware of that. Will the version stay the same? It looks like the majority of the pil providers are different versions of py-pillow.

Reply:
Yes, versions are compatible. I always use py-pillow-simd since it's much faster.